frontend/assets/stylesheets/components/_listings.scss

```diff
@@ -2,6 +2,27 @@
    Listings
    ========================================================================== */
 
+.listing-header {
+    background-color: $c-background;
+    padding: rem($gs-gutter / 2);
+
+    @include mq(mem-full) {
+        padding-left: rem(gs-span(2) + ($gs-gutter * 2));
+        padding-bottom: rem($gs-gutter * 4);
+    }
+}
+
+.listing-headline {
+    @include fs-headline(4);
+
+    @include mq(tablet) {
+        @include fs-headline(5);
```

Comment:
Pass `@include fs-headline(5, true)` to avoid generating duplicate font-stacks here. Not a big deal as gzip chews through this kind of repetition, but worth avoiding duplicating styles if we can.